src/uwtools/scheduler.py

```diff
@@ -284,14 +309,22 @@ class Slurm(JobScheduler):
     Represents the Slurm scheduler.
     """
 
+    @property
+    def _forbidden_directives(self) -> list[str]:
+        """
+        Returns directives that this scheduler does not support.
+        """
+        return [_DirectivesOptional.SHELL]
+
     @property
     def _managed_directives(self) -> dict[str, Any]:
         """
         Returns a mapping from canonical names to scheduler-specific CLI switches.
         """
         return {
             _DirectivesOptional.CORES: "--ntasks",
-            _DirectivesOptional.EXCLUSIVE: lambda _: "--exclusive",
+            _DirectivesOptional.DEBUG: lambda b: "--verbose" if b else None,
+            _DirectivesOptional.EXCLUSIVE: lambda b: "--exclusive" if b else None,
```

Comment:
For these boolean switches, return `None` if the switch should not be specified at all.